src/App.js

```diff
@@ -1,45 +1,24 @@
+import { Routes, Route } from "react-router-dom";
+import {Cart, Wishlist, Home, Login, Filter, Product, Navbar} from "./Components/index"
+
```

Comment:
here in the import from `./Components/index` , `/index` can be ignored as index.js will be automatically picked up.